src/NuGet.Indexing/ResponseFormatter.cs

```diff
@@ -96,6 +97,35 @@ private static void WriteData(JsonWriter jsonWriter, NuGetIndexSearcher searcher
             jsonWriter.WriteEndArray();
         }
 
+        private static void WritePackageTypes(JsonWriter jsonWriter, Document document)
```

Comment:
There are currently no UTs (as far as i can tell) for the responseFormatter in general other than the V2/V3 functional tests. I propose that we add these as part of a different PR expanding our testing of this class in general.

---
In reply to: [78818136](https://github.com/NuGet/NuGet.Services.Metadata/pull/87#discussion_r78818136) [](ancestors = 78818136)